.github/workflows/snakemake-ci.yml

```diff
@@ -0,0 +1,65 @@
+name: Snakemake CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
```

Comment:
So as the environment file is already defining what Python to be used

https://github.com/AndriiPovsten/Snakemake-backend-for-RECAST/blob/03ad8f05dd2ae15fa504deb0964b511ab84d6660/recast_helloworld/environment.yml#L4-L5

you don't need to setup a Python runtime here